From 41f944c211cc720b77d70348cf7cf2ad9fd3c135 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Wed, 15 Nov 2017 15:07:32 +0800
Subject: [PATCH 2085/2241] arm64: dts: a80x0: disable serial interface of CPs

CP serial interfaces are not used in development boards,
this patch disables the serials in CPs.

Change-Id: I8b9ca4932be4d06b394f822eaf7c6b449bea971c
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/46394
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/46393
---
 arch/arm64/boot/dts/marvell/armada-8040-db-B.dts           | 3 ---
 arch/arm64/boot/dts/marvell/armada-8040-db-C.dts           | 3 ---
 arch/arm64/boot/dts/marvell/armada-8040-db-E.dts           | 3 ---
 arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts | 3 ---
 arch/arm64/boot/dts/marvell/armada-8040-db.dts             | 3 ---
 5 files changed, 15 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
index fd4e4c8..85dfea6 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
@@ -165,9 +165,6 @@
 				phy-names = "usb2", "usb3";
 				status = "okay";
 			};
-			serial@702000 {
-				status = "okay";
-			};
 			sata@540000 {
 				status = "okay";
 				sata-port@1 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
index 1534c22..6c6cfcb 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
@@ -178,9 +178,6 @@
 				phys = <&cp1_utmi0>;
 				phy-names = "usb";
 			};
-			serial@702000 {
-				status = "okay";
-			};
 			sata@540000 {
 				status = "okay";
 				sata-port@0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
index 64695d00..68fa453 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
@@ -181,9 +181,6 @@
 				phys = <&cp1_utmi0>;
 				phy-names = "usb";
 			};
-			serial@702000 {
-				status = "okay";
-			};
 			sata@540000 {
 				status = "okay";
 				sata-port@0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
index b4a0bf2..85552b7 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
@@ -187,9 +187,6 @@
 			cp1_usb3h0: usb3@500000 {
 				status = "okay";
 			};
-			serial@702000 {
-				status = "okay";
-			};
 			sata@540000 {
 				status = "okay";
 				sata-port@0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dts b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
index 0f5a4b2..0443cb0 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
@@ -179,9 +179,6 @@
 				phys = <&cp1_utmi0>;
 				phy-names = "usb";
 			};
-			serial@702000 {
-				status = "okay";
-			};
 			sata@540000 {
 				status = "okay";
 				sata-port@0 {
-- 
2.7.4

